.npmignore

```diff
@@ -7,6 +7,8 @@ obj
 *.sln
 *.njsproj
 .vscode/
+.github
+applicationinsights.json

```

Comment:
We want this file to be part of npm package, it is available for configuration of the SDK

Reply:
Done